app/src/main/res/layout/fragment_pair.xml

```diff
@@ -0,0 +1,22 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
nit - this formatting looks a little wonky. 

If you go to "Android Studio" -> Preferences -> Editor -> Code Style -> XML -> "Set from..." -> Predefined Style -> Android 
and then Code -> Reformat it will do it for you :)